docs/resource-policy/installation.md

```diff
@@ -1 +1,264 @@
-# Installation
+# Installation methods
+
+This repository hosts a collection of plugins of various types, one of which is the resource
+policy plugins. In this example, we will demonstrate the installation process for the topology-aware
+plugin, which falls under the resource policy type. The installation methods outlined
+here can be applied to any other plugin hosted in this repository, regardless of its type.
+
+You have the option to choose either of these installation methods to install the NRI plugin into
+your cluster. 
+
+1. [Helm](#Helm)
+2. [Kustomize](#Kustomize)
+3. [Manual](#Manual)
+
+Regardless of the chosen installation method, the NRI plugin installation includes the
+following components: DaemonSet, ConfigMap, CustomResourceDefinition, and RBAC-related objects.
+
+### Prerequisites
+
+Container runtime:
+- containerD:
+    - At least [containerd 1.7.0](https://github.com/containerd/containerd/releases/tag/v1.7.0) release version to use the NRI feature
+    - Enable NRI feature by following [these](TODO link) detailed instructions.
+- CRI-O
+    - At least [v1.26.0](https://github.com/cri-o/cri-o/releases/tag/v1.26.0) release version to use the NRI feature
+    - Enable NRI feature by following [these](TODO link) detailed instructions.
+
+## Helm
+
+1. Clone the project to your local machine
+    ```sh
+    git clone https://github.com/containers/nri-plugins.git
+    ```
+
+1. Navigate to the project directory
+    ```sh
+    cd nri-plugins
+    ```
+
+1. Install the plugin using Helm. Replace release name with the desired name for your Helm release. In this example, we named it as topology-aware. The default values for topology-aware resource policy plugin are stored in [topology_aware_values.yaml](TODO) file. If you wish to provide custom values to the Helm chart, refer to the [table](#Helm-default-values) below, which describes the available parameters that can be modified before installation.
+
+    ```sh
+    helm install -f deployment/helm/nri-plugins/topology_aware_values.yaml topology-aware ./deployment/helm/nri-plugins/
+    ```
+
+1. Verify the status of the daemonset to ensure that the plugin is running successfully
+
+    ```terminal
+    kubectl get daemonset -n kube-system nri-resource-policy
+    
+    NAME                  DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE
+    nri-resource-policy   1         1         0       1            0           kubernetes.io/os=linux   4m33s
+    ```
+
+That's it! You have now installed the topology-aware NRI resource policy plugin using Helm.
+
+#### Helm default values
+
+This table provides an overview of the default values for the Topology-aware plugin Helm chart.
+
+<table>
+<tr>
+<td>
```

Comment:
Instead of embedding html code to this file, could you use the markdown table formatting here?

Reply:
Fixed